pkgs/by-name/bo/boosteroid/package.nix

```diff
@@ -0,0 +1,117 @@
+{
+  stdenv,
+  lib,
+  dpkg,
+  fetchurl,
+  autoPatchelfHook,
+  makeWrapper,
+  xorg,
+  libva-minimal,
+  xz,
+  libva,
+  libvdpau,
+  libxkbcommon,
+  wayland,
+  libgcc,
+  freetype,
+  systemd,
+  alsa-lib,
+  fontconfig,
+  pcre2,
+  dbus,
+}:
+stdenv.mkDerivation {
+  name = "boosteroid";
+  version = "1.8.4";
+
+  # They don't provide a way to download a specific version, and I didn't find developer contacts yet.
+  src = fetchurl {
+    url = "https://boosteroid.com/linux/installer/boosteroid-install-x64.deb";
```

Comment:
This is problematic, as every user has to download this from upstream instead of `cache.nixos.org` (because we can't host unfree binaries), causing build failures whenever upstream decides to update the binary.

Reply:
Yes, but since they also have a HARD latest version check within program itself, I'd consider that «early failover» :D

Reply:
Oh it's always forced to stay on latest? That could work as long as people update it in Nixpkgs as soon as possible, but it's still not ideal considering it still takes a few days for master to propagate to unstable and then to specific release branches like 23.11 and 24.05.

Reply:
My idea was to implement something akin to what some vscodium extensions do — download updates and patch them in place. However that will probably be «boosteroid-installer»

Reply:
Using an unstable source is not going to pass for nixpkgs. You need a stable mirror.